index.html

```diff
@@ -169,10 +169,27 @@ <h5 class="media-text">LET’S CONNECT</h5>
                     <img src="./assets/Normal Button/Tertiary/Icons/Icon - Chevron - Right - Larger.svg">
                 </li>
                 <div class="line"></div>
-            </ul>
+
+            </div>
         </div>
 
     </section>
+    <section class="contact-section">
+        <h2 class="contact-heading">Contact me</h2>
+        <p class="contact-cta">If you have an application you are
+            interested in developing, a feature
+            that you need built or a project that
+            needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/{form_id}" class="contact-form" method="post">
+            <input type="text" id="name" placeholder="Yeremaias" maxlength="30" required />
+            <input type="email" placeholder="natajayanj14@gmail.com" id="email" required />
+            <textarea name="message" id="message" cols="25" rows="10" maxlength="500"
+                placeholder="Write your message here" required></textarea>
+            <button id="submit" type="submit">Get in Touch</button>
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly implement the button interactions of `hover`, `enable` and `active` on this button.